packages/slate-editor/src/modules/editor/Editor.tsx

```diff
@@ -574,7 +591,7 @@ export const Editor = forwardRef<EditorRef, EditorProps>((props, forwardedRef) =
                     <FloatingStoryEmbedInput
                         availableWidth={availableWidth}
                         containerRef={containerRef}
-                        onClose={closeFloatingStoryBookmarkInput}
+                        onClose={closeFloatingSnippetInput}
```

Comment:
@e1himself I was browsing this PR for reference, and noticed this change, which seems incorrect. I believe it should've not been touched.

Reply:
good catch, i will fix, thanks

Reply:
PR: https://github.com/prezly/slate/pull/318